frontend/src/locales/scripts/en.json5

```
@@ -437,7 +437,36 @@
       creator: "search by creator",
       imageTitle: "Image: {title}",
       audioTitle: "Audio: {title}",
-      results: "Search results for {query}.",
+      results: {
+        /**
+         * "imageResults" and "audioResults" are interpolated from the strings under
+         * browsePage.aria.allResultsHeadingCount.*
+         */
+        all: "All results for {query}, {imageResults} and {audioResults}.",
+        image: {
+          zero: "No image results for {query}",
+          count: "{localeCount} image result for {query}.|{localeCount} image results for {query}.",
+          countMore: "Over {localeCount} image results for {query}.",
+        },
+        audio: {
+          zero: "No audio results for {query}",
+          count: "{localeCount} audio result for {query}.|{localeCount} audio results for {query}.",
+          countMore: "Over {localeCount} audio results for {query}.",
+        },
+      },
+      /** These are interpolated into browsePage.aria.results.all */
+      allResultsHeadingCount: {
+        image: {
+          zero: "no images",
+          count: "{localeCount} image|{localeCount} images",
+          countMore: "over {localeCount} images",
+        },
+        audio: {
+          zero: "no audio",
+          count: "{localeCount} audio",
+          countMore: "over {localeCount} audio",
```

Subjective opinion: Number with 'audio' felt wrong.

```suggestion
          count: "{localeCount} audio tracks",
          countMore: "over {localeCount} audio tracks",
```

---

Done in https://github.com/WordPress/openverse/pull/3941/commits/db46a043bcfe4281b2f9329f30436c0b82c74cd3

---

@sarayourfriend I believe "tracks" is still missing in the cases where the string is `All results for "zack", 10 images and 10 audio`.